### PR TITLE
about card metric image tag removing /

### LINKED ISSUE #5253: img Tag Refactor: _includes/about-page/about-card-metrics.html - metrics.svg
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started) and attend an onboarding session.
2. Before you claim or start working on an issue, please make sure you have read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/7f0c132c96f71230b8935759e1f8711ccb340c0f/CONTRIBUTING.md).

### Overview
We want to change an img HTML tag ending with a slash () to an img tag without an ending slash () so that the codebase is consistent with how we use img HTML tags.

### Action Items
- [x] In your local IDE, navigate to `_includes/about-page/about-card-metrics.html`
- [x] Remove the ending slash in the img HTML tag by changing
```
<img src="/assets/images/about/section-header-elements/metrics.svg" alt="" />
```
to
```
<img src="/assets/images/about/section-header-elements/metrics.svg" alt="">
```
- [x] The file may contain more than one instance of an img tag with a closing slash.  Only change the single instance specified above.  Do not leave any blank spaces before the closing bracket.
- [x] Using Docker, check the page remains the same in mobile, tablet, and desktop views as on the current website (See 2 in the Resources/Instructions section below)

### Merge Team
- [ ] When this issue is completed, check off the issue in #5040

### Resources/Instructions
1. For QA to validate change: https://github.com/hackforla/website/blob/gh-pages/_includes/about-page/about-card-metrics.html
2. Webpage(s): https://www.hackforla.org/about/
3. This issue is tracked in [HfLA-Website: img Tag Audit spreadsheet's img with Ending Slash tab](https://docs.google.com/spreadsheets/d/1cq8iLOYQkAbho2GsRAxVJYgjxDeJ9KLdx2NuL4G6vu4/edit#gid=0), and #5040

### PR DESCRIPTION
**Pull request by Jfong1218**
Fixes #5253

### What changes did you make?
 - Removed ending slash for img tag of metrics.svg

### Why did you make the changes (we will use this info to test)?
  - Needed a consistent codebase with how we use img HTML tags

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

no visual change

### CODE REVIEW

**Comment by meru-a** [comment]
Availability: In the evening & on weekends
ETA: EOD 08/27

File: _includes/about-page/about-card-metrics.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
         <div class="flex-container">
             <!--This is the top and then right cell-->
             <div class="top-left-column">
-                <div class="small-card">
+                <div class="small-card">w
```
Comment (meru-a) [inline _includes/about-page/about-card-metrics.html]: @Jfong1218 Here is the line with an extra character at the end. Please remove the character and re-submit the changes, thank you!

**Review by meru-a — CHANGES_REQUESTED** [review]
- The new branch is properly going into the correct branch gh-pages. ✅
- Original issue was properly linked and referenced. ✅
- The specified instance of an `` tag was properly changed. ✅
- No additional changes were made. ❌

Great work! However, on the line 16 of file `_includes/about-page/about-card-metrics.html` there is an extra character 'w' that shouldn't be there. After this small edit, I'll be glad to approve your pull request!

**Comment by Jfong1218** [comment]
Fixes #5253

### What changes did you make?
Removed ending slash for img tag of metrics.svg
also removed w after line 16 
### Why did you make the changes (we will use this info to test)?
Needed a consistent codebase with how we use img HTML tags
accidentally typed w on line 16
### Screenshots of Proposed Changes Of The Website (if any, please do not screen shot code changes)
No visual changes

**Comment by adrianang** [comment]
> Fixes #5253
> 
> ### What changes did you make?
> Removed ending slash for img tag of metrics.svg also removed w after line 16
> 
> ### Why did you make the changes (we will use this info to test)?
> Needed a consistent codebase with how we use img HTML tags accidentally typed w on line 16
> 
> ### Screenshots of Proposed Changes Of The Website (if any, please do not screen shot code changes)
> No visual changes

Hi @Jfong1218 — you'll want to _edit the original post_ of your PR (go to the top-right of your original post in this PR and click 'Edit' in the dropdown) with these updates instead. Also, you don't need to make note of the fix to the typo you made in the PR; the PR should only document whatever will get merged into the HfLA Website codebase (so bullets for just the removal of the ending slash, as the typo wouldn't get merged in).

**Review by meru-a — APPROVED** [review]
The extra character seems to be removed, so I'm approving the request. Great job!

**Comment by kwangric** [comment]
Hey @DorianDeptuch! When you get the chance, please leave a comment with an availability and ETA for your review. Thanks!

**Comment by DorianDeptuch** [comment]
Availability: 8/28 Afternoon
ETA: 8/28 EOD

**Comment by Jfong1218** [comment]
> > Fixes #5253
> > ### What changes did you make?
> > Removed ending slash for img tag of metrics.svg also removed w after line 16
> > ### Why did you make the changes (we will use this info to test)?
> > Needed a consistent codebase with how we use img HTML tags accidentally typed w on line 16
> > ### Screenshots of Proposed Changes Of The Website (if any, please do not screen shot code changes)
> > No visual changes
> 
> Hi @Jfong1218 — you'll want to _edit the original post_ of your PR (go to the top-right of your original post in this PR and click 'Edit' in the dropdown) with these updates instead. Also, you don't need to make note of the fix to the typo you made in the PR; the PR should only document whatever will get merged into the HfLA Website codebase (so bullets for just the removal of the ending slash, as the typo wouldn't get merged in).

**Review by DorianDeptuch — COMMENTED** [review]
Hi @Jfong1218,

I ran this in my local environment and everything seems to work well. You removed the ending img slash correctly. In the discussion, you have the What and the Why sections done correctly and there are no visual changes. I would consider editing your post to remove the `<details>` from the visual changes section and instead writing something along the line of "No visual changes." You can follow @adrianang 's comment in the conversation to find the instructions to edit your post. It is not necessary to mention anything about removing the "w" in the What or the Why sections because it does not pertain to the original issue.

Besides that, everything looks good to me. Great job. Re-request me for a review when you make the changes and I will approve the PR.

**Comment by Jfong1218** [comment]
Hello Team,

I've made the necessary changes in my PR per @DorianDeptuch and @adrianang request and have deleted the duplicate comment.

Thanks!

**Comment by christinaor** [comment]
Hi @Jfong1218 , good job on this issue! It looks like you made the necessary changes based on the feedback in comments. I will merge this, but in the future, you may request a Hack for LA member to review your issue again by clicking on the revolving circle next to their username (see the red square in the image):

image

**Review by christinaor — APPROVED** [review]

**Review by christinaor — APPROVED** [review]

File: _includes/about-page/about-card-metrics.html (hunk 1 above)
Comment (christinaor) [reply _includes/about-page/about-card-metrics.html]: The extra character has been resolved in the latest changes.

**Pull request merged**